bitfield/impl/src/define_specifiers.rs

```diff
@@ -0,0 +1,39 @@
+use proc_macro::TokenStream;
+use proc_macro2::Span as Span2;
+use syn;
+use quote::{
+    quote,
+};
+
+pub fn generate(_input: TokenStream) -> TokenStream {
+    let mut tokens = quote!{};
+    for n in 1..=64 {
```

Comment:
I didn't handle `u128` in the rest of the specification because it wasn't specified.
Also I am not sure it is worth supporting it because it might increase compile times substantially.
I was thinking that I would rather implement `u128` support behind a crate feature.